contracts/Token.sol

```diff
@@ -149,11 +159,11 @@ contract GolemNetworkToken {
         Transfer(0, msg.sender, numTokens);
     }
 
-    // If cap was reached or crowdfunding has ended then:
-    // create GNT for the Golem Factory and developer,
-    // transfer ETH to the Golem Factory address.
-    // Required state: Funding Success
-    // State transition: -> Operational Normal
+    /// @dev If cap was reached or crowdfunding has ended then:
```

Comment:
Add introducing sentence, e.g. "Finalize crowdfunding." Keep the rest in `@notice`.